modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/capability/xml/schema/SpringHandlerBundleResourceFactory.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.internal.capability.xml.schema;
+
+import org.mule.runtime.extension.api.introspection.ExtensionModel;
+import org.mule.runtime.extension.api.introspection.property.XmlModelProperty;
+import org.mule.runtime.extension.api.resources.GeneratedResource;
+import org.mule.runtime.module.extension.internal.config.ExtensionNamespaceHandler;
+
+/**
+ * Implementation of {@link AbstractXmlResourceFactory} which generates a
+ * Spring bundle file which links the extension's namespace to the {@link ExtensionNamespaceHandler}
```

Comment:
Generates a Spring bundle file which links the extension's namespace to the {@link ExtensionNamespaceHandler}